### DISCUSSION TITLE
Is React a front end or backend?

### DISCUSSION THREAD

**Discussion by Sachin066** [community/community#67731] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

People still wonder if React Js JS is frontend or backend?

**Comment by Rinki2002** [comment]
No doubt, React Js is a front-end JavaScript library used in web development to build interactive elements on website User Interfaces (UI) based on UI components.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be closing and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
